.travis.yml

```diff
@@ -18,3 +18,8 @@ deploy:
   on:
     repo: SUSE/saltstack-netapi-client-java
     tags: true
+after_success:
+- scripts/publish_javadoc.sh
+env:
+  global:
+  - secure: aFeCOD180jlpw9yCqyKDXDuMrcVaNj607sCsIcFZ+rIYo1bMFWl5iLrQasQgtzE9jtNto6VYRq/BHWDR7qBSyk0qg2yWh+qhs7O/53m1XDD5DJryVg89IW4ynauCIeVNSfaLRzWRklphHeaLifNxBLd0v9hAG6z+skx65s7VzLM=
```

Comment:
OK @lucidd, here is the new encrypted token (`travis encrypt <token>`):

    secure: "braT4pMYLy0ocRgNt4MrI0kWUPnQF455RNLXBAf3+ArSiCMpaoQkwoDnauI155+Kaq2sXDuFg/vCdNQ0h4Rd33qEW5vg810wQQKj9lleDJmfv2W6/DxpTDnvySR4ChAWv9BZBMFIPhdv/dfZWgZBFHluvXw2Bi4UMF0GXsNPjYs="

And this is the encrypted environment variable `GH_TOKEN` (`travis encrypt GH_TOKEN=<token>`):

    secure: "XvflMMZoyQSbsCdOD98rFxsEvd+c9aNUulPf0IIKpp6OFcl0lEoq6c4cEd0C/qBtlZgBNTYmnWfOpWcrNZABa2PXAbi0p9lw/5/L9UgnlEfWG0NbVbJ/arb2DidgnidvVT5GhNNn31/38zwVcJVyo3yjRmf72ZwPbZKWGygKkjc="